src/octoprint/util/comm.py

```diff
@@ -429,6 +429,21 @@ def _to_new_tuple(cls, current, actual, target):
             return actual, target
 
 
+class FanspeedRecord:
+    def __init__(self):
+        self._fanspeed = None
+
+    def copy_from(self, other):
+        self._fanspeed = other.fanspeed
+
+    def set_speed(self, speed):
+        self._fanspeed = speed
+
+    @property
+    def fanspeed(self):
+        return self._fanspeed
+
+
```

Comment:
No need for that :) For the temperature records, we do this since we have to keep track of actual & targets of multiple heaters. Here we only track one single integer value, so we might as well use that instead of a our new type.